gcc-4.8/post-link.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+
+if [ "$(uname)" != "Darwin" ]; then
```

Comment:
Hmm, conda-build really ought to have a way to not even include the file on OS X.